crates/re_viewer_context/src/gpu_bridge/tensor_to_gpu.rs

```diff
@@ -27,6 +27,30 @@ use super::{get_or_create_texture, try_get_or_create_texture};
 
 // ----------------------------------------------------------------------------
 
+#[derive(Copy, Clone)]
```

Comment:
```suggestion
#[derive(Copy, Clone, Hash)]
```

Reply:
oh. wow. of course. I started out with fixing a single value and then ended up creating this enum without thinking to hash it